Specification.md

```diff
@@ -110,25 +111,34 @@ A *range* is an *entry* that represents the time span between two points in time
 
 It MUST consist of two values that denote the start and the end.
 Start and end MUST be written in chronological order.
+They MAY be equal.
 
 There MUST be a `-` between the two values.
 There MAY appear “spaces” on either side of the `-`;
 for this case it is RECOMMENDED to use exactly one “space” on both sides.
```

Comment:
I am not fully versed in the English grammar, but I do not believe you should use "for this case" after a semicolon.

Could you take a second look at this and see if you think this is valid? (I might be completely wrong here)

Reply:
(I used Google Translate to find the phrase "I am not fully versed in")

Reply:
Hm, that never occurred to me. Why do you think that could be invalid?

I changed `for` for `in`, though, because I think that’s way more common. What do you think of:

> There MAY appear “spaces” on either side of the `-`,
> in which case it is RECOMMENDED to use exactly one “space” on both sides.

Reply:
My reasoning is that `for this case it is RECOMMENDED to use […]` could not stand as its own sentence.

This is one of the grammatical nitpicks I mentioned. This change would not help understanding the content of the spec, so I feel no real urge to change it.

Reply:
Ah good point with the standalone sentence. I’m not fully versed in english grammer either (except for the usage of sophisticated expressions like “to be fully versed in”), so the proposed version would conveniently solve this problem. (https://github.com/jotaen/klog/pull/88/commits/ec39ccc64004998d38f1c5905d6e465de7586b5f)